internal/gh/pullrequest.go

```diff
@@ -24,6 +24,20 @@ type PullRequest struct {
 	Permalink   string
 	State       githubv4.PullRequestState
 	Title       string
+	MergeCommit struct {
```

Comment:
Can you change this from `MergeCommit struct {...}` to `` PRIVATE_MergeCommit struct {..} `graphql:"mergeCommit"` ``? Just to indicate to calling code that we really shouldn't be accessing it directly.

Then renaming the function to just `MergeCommit` is good.

Reply:
Similar for `TimeLineItems` -- also lets do `PRIVATE_TimelineItems` there (no capital `L` to match GitHub here)